### PR TITLE
feat: Add RR crossing number to updated mobile and desktop menus

### PR DESCRIPTION
**Pull request by bfauble**
#### Summary of changes
**Asana Ticket:** [Add RR crossing number to updated mobile and desktop menus](https://app.asana.com/0/385363666817452/1201824241791486/f)

Add the railroad crossing gate issue number to the new desktop and mobile menus.

image
image
image

### CODE REVIEW

File: apps/site/lib/site_web/views/helpers.ex [hunk 1]
```diff
@@ -264,6 +264,7 @@ defmodule SiteWeb.ViewHelpers do
 
       machine_formatted ->
         content_tag(:a, pretty_formatted,
+          class: "no-wrap",
```
Comment (thecristen) [inline apps/site/lib/site_web/views/helpers.ex:267]: 👍🏼 Good addition.

**Review by thecristen — CHANGES_REQUESTED** [review]
💯 on mobile/tablet view, thanks for the attention to white space/text wrapping.

The mock for the desktop view structures this new section a bit differently, placing it in the bottom of the center column instead of below the two sections in the right column. Can you update this to reflect that too?

**Comment by bfauble** [comment]
https://www.w3.org/TR/css-grid-1/#repeat-syntax
https://www.w3.org/TR/css-grid-1/#valdef-repeat-auto-fit

couple relevant pieces of info regarding generating auto-fit columns

**Review by thecristen — APPROVED** [review]
Great!

**Pull request merged**